src/nvim/os/fs.h

```diff
@@ -2,8 +2,21 @@
 #define NVIM_OS_FS_H
 
 #include "nvim/os/fs_defs.h"
+#include "nvim/os/os_defs.h"
 #include "nvim/types.h"
 
+#if defined(HAVE_ACL)
+# define OS_NEW_ACL(aclent) vim_acl_T aclent = NULL
+# define OS_GET_ACL(aclent, fname) { (aclent) = os_get_acl(fname); }
+# define OS_FREE_ACL(aclent) os_free_acl(aclent)
+# define OS_SET_ACL(fname, aclent) os_set_acl((fname), (aclent))
+#else
+# define OS_NEW_ACL(aclent)
+# define OS_GET_ACL(aclent, fname)
+# define OS_FREE_ACL(aclent)
+# define OS_SET_ACL(fname, aclent)
+#endif
+
```

Comment:
`os_get_acl()` and `os_set_acl()` are actually always no-op, so this just adds another layer of redundant wrapper.

Reply:
Maybe it's better to make these functions always available?

Reply:
If they are always no-op, then why do they exist?

Reply:
For future implementation, I guess.

Reply:
ok fair enough. Let's just expose them.